### PR TITLE
v1.22.0

### PR DESCRIPTION
**Pull request by JelteF**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,6 +1,38 @@
...
+
+- Changes
+  * Improve settings in the recommended systemd unit file ([#983])
+  * Make fail fast logic handle all scenarios where no working connections to
+    the database exist anymore and none can be established ([#998])
+  * Multiple documentation improvements
+
+- Fixes
+  * Fix issue in PG14+ where PgBouncer would send `SET DateStyle='ISO'` for
+    every transaction. ([#879])
```
Comment (emelsimsek) [inline NEWS.md:23]:
```suggestion
    every transaction ([#879])
```

File: NEWS.md [hunk 2]
```diff
@@ -1,6 +1,38 @@
...
 ===================
 
+PgBouncer 1.22.x
+----------------
+
+**XXXX-YY-ZZ  -  PgBouncer 1.22.0  -  "DEALLOCATE ALL"
+
+- Features
+  * Adds support for `DEALLOCATE ALL` and `DISCARD ALL` when
+    `max_prepared_statements` is set to a non-zero value. (Normal `DEALLOCATE`
```
Comment (emelsimsek) [inline NEWS.md:11]:
```suggestion
    `max_prepared_statements` is set to a non-zero value (Normal `DEALLOCATE`
```

**Review by emelsimsek — APPROVED** [review]

**Comment by JelteF** [comment]
> LGTM. Maybe you should include commit [6c7730e](https://github.com/pgbouncer/pgbouncer/commit/6c7730e2e9f2a5710292a5535e17dbb8882679ff) to get the attention of packagers.

It's in the changelog as the first item in the "Changes" section of the release:
```
  * Improve settings in the recommended systemd unit file ([#983])
```

Did you miss that line? Or did you mean we should call it out more than that?

**Comment by eulerto** [comment]
I clearly missed that. ENOCOFFEE.

**Review by eulerto — APPROVED** [review]

**Pull request merged**